### ISSUE TITLE
question about the model of  mobilenet_fast

### ISSUE THREAD

**Issue by andyqian2015** [ildoonet/tf-pose-estimation#36]
Hi,@ildoonet  .I success in running tf-pose-estimation on the embedding device using the model of mobilenet. but the FPS is limited to less than 1.I try to check the model of mobilenet_fast.but find nothing.I beg if you can share the model of mobilenet_fast or dsecription of training the mobilenet-fast.I would be appreciated if you can reply. Thx  ildoonet.

**Comment by thundertrick** [comment]
The project has a pretrained checkpoint model for mobilenet_fast, but it seems we can not use it directly according to issure #34.

**Comment by ildoonet** [comment]
@andyqian2015 

I have a plan to release variants of the default model. 

But for now, it is quiet not satisfactory with respect to its speed and accuracy. 

I'm trying to improve them as much as I can. I will keep you updated.

**Comment by andyqian2015** [comment]
thx for your reply @ildoonet ,waiting for your update.

**Comment by GoForPeak** [comment]
I have completed the paper:Distilling the Knowledge in a Neural Network  to improve the speed,but the accuracy is not better than before. When I test it on 1050ti with 368*368 and 656*368 ,it cost about 75ms and 110ms. I have tried to implement the mobilenetv1-v2 on caffe ,but  it cannot be tarined since the caffe's memory management is no efficiency. I will try to use your tf version, I hope it works amazing

**Comment by GoForPeak** [comment]
I have completed the paper:Distilling the Knowledge in a Neural Network  to improve the speed,but the accuracy is not better than before. When I test it on 1050ti with 368*368 and 656*368 ,it cost about 75ms and 110ms. I have tried to implement the mobilenetv1-v2 on caffe ,but  it cannot be tarined since the caffe's memory management is no efficiency. I will try to use your tf version, I hope it works amazing

**Comment by ouceduxzk** [comment]
there are many knowledge distillation methods, @GoForPeak  do you try something else ?  I tried with a pruned vgg16 model, it works nicely, but not as fast as this mobilenet_thin model, however more accurate than mobilenet_thin

**Comment by GoForPeak** [comment]
@ouceduxzk, thank you for reminding me about this, I have not tried other methods now, I hope that the accuracy vs speed trade off, your pruned vgg16 model works good, how about the time of per frame forward process?         if the time is enough for me,  I will try the paper :Deep Compression Compressing Deep Neural Networks with Pruning Trained Quantization and Huffman coding,  it may works.
